cms/templates/manage_users.html

```diff
@@ -97,6 +97,7 @@ <h3 class="title-3">${_("Course Team Roles")}</h3>
         <p>${_("Course team members with the Staff role are course co-authors. They have full writing and editing privileges on all course content.")}</p>
         <p>${_("Admins are course team members who can add and remove other course team members.")}</p>
         <p>${_("All course team members are automatically enrolled in the course and can access content in Studio, the LMS, and Insights.")}</p>
+      ## Note that this "Admin" role is identified as "Instructor" in the Django admin panel.
```

Comment:
The syntax looks right.  Inserting this above line 98 help developers see it where the clarity is needed.